src/backend/app.js

```diff
@@ -21,10 +31,50 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//testing knex connection
+router.get("/info", async (req, res) => {
+  const dbResult = await knex.raw("SELECT VERSION()");
+  const row = dbResult[0][0];
+  res.json({ nodeVersion: process.version, mysqlVersion: row["VERSION()"] });
+});
+
+router.get("/future-meals", async (req, res) => {
+  const [rows] = await knex.raw("SELECT * from meal WHERE `when` > CURDATE()");
```

Comment:
Where condition should be `>=` for `future meals`

Reply:
It is good idea to give descriptive names to understand the purpose of that variable.

Reply:
Not suggested to select `*`, instead select whatever specifically you want from the table like, `title`/`price`/etc.

Reply:
> Where condition should be `>=` for `future meals`

changed

Reply:
> Not suggested to select `*`, instead select whatever specifically you want from the table like, `title`/`price`/etc.

changed to title